backend/src/main/java/de/neuefische/ffmjava221/teamprojekt/backend/employee/EmployeeDTO.java

```diff
@@ -1,7 +1,15 @@
 package de.neuefische.ffmjava221.teamprojekt.backend.employee;
 
-public record UpdateEmployee(
+import org.springframework.data.annotation.Id;
 
+import javax.validation.constraints.NotBlank;
+
+public record EmployeeDTO(
+        @Id
```

Comment:
Since you never save the dto, there is no need to annotate the id

Reply:
ok